CHANGELOG.md

```
@@ -0,0 +1,1226 @@
+[v1.19.1]: https://github.com/capcom6/android-sms-gateway/compare/v1.19.0...v1.19.1
+[v1.19.2]: https://github.com/capcom6/android-sms-gateway/compare/v1.19.1...v1.19.2
+[v1.20.0]: https://github.com/capcom6/android-sms-gateway/compare/v1.19.2...v1.20.0
+[v1.21.0]: https://github.com/capcom6/android-sms-gateway/compare/v1.20.0...v1.21.0
+[v1.21.1]: https://github.com/capcom6/android-sms-gateway/compare/v1.21.0...v1.21.1
+[v1.22.0]: https://github.com/capcom6/android-sms-gateway/compare/v1.21.1...v1.22.0
+[v1.23.0]: https://github.com/capcom6/android-sms-gateway/compare/v1.22.0...v1.23.0
+[v1.23.1]: https://github.com/capcom6/android-sms-gateway/compare/v1.23.0...v1.23.1
+[v1.24.0]: https://github.com/capcom6/android-sms-gateway/compare/v1.23.1...v1.24.0
+[v1.25.0]: https://github.com/capcom6/android-sms-gateway/compare/v1.24.0...v1.25.0
+[v1.26.0]: https://github.com/capcom6/android-sms-gateway/compare/v1.25.0...v1.26.0
+[v1.27.0]: https://github.com/capcom6/android-sms-gateway/compare/v1.26.0...v1.27.0
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "v1.27.0"

(MD053, link-image-reference-definitions)